app/controllers/api/v1/topics_controller.rb

```diff
@@ -0,0 +1,11 @@
+module Api
+  module V1
+    class TopicsController < ApplicationController
+      before_action :authenticate_user!
+      
+      def index
+        @topics = Topic.all
```

Comment:
It's not a problem here, but always when you do `Something.all` you should think about using pagination or not.